images/utapi/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12-alpine
```

Comment:
Can we move to Node 12?

I support using node12 due to its nice new features. But I was told that we are going to maintain all components on v10 LT for now.

May be, @rahulreddy  can confirm.

Reply:
I went with 12 because 10 EOL's on 05/2021. I don't think starting a new project using 10 makes sense as we'd have to upgrade in about a year anyway